packages/gatsby-plugin-sass/package.json

```diff
@@ -8,15 +8,15 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
-    "extract-text-webpack-plugin": "^1.0.1",
```

Comment:
The helper handles this dependency so plugins utilizing the helper don't need it.